tmpfiles.d/systemd-pstore.conf

```diff
@@ -0,0 +1,27 @@
+#
+#The systemd pstore service archives the contents of /sys/fs/pstore
+#upon boot so that there is room for a subsequent dump. This service
+#is enabled with:
+#  systemctl enable systemd-pstore
+#
+#With the service enabled, the kernel still needs to be configured
+#to write data into the pstore. The kernel has two parameters,
+#crash_kexec_post_notifiers and printk.always_kmsg_dump, that
+#control writes into pstore.
+#
+#The crash_kexec_post_notifiers parameter enables the kernel to write
+#dmesg (including stack trace) into pstore upon a panic, and
+#printk.always_kmsg_dump parameter enables the kernel to write dmesg
+#upon a shutdown (shutdown, reboot, halt).
+#
+#To configure the kernel parameters, uncomment the appropriate
+#line(s) below. The value written is either 'Y' to enable the
+#kernel parameter, or 'N' to disable the kernel parameter.
+#
+#After making a change to this file, do:
+#  systemd-tmpfiles --create path/to/tmpfiles.d/systemd-pstore.conf
+#
+#These changes are automatically applied on future re-boots.
+
+#w /sys/module/printk/parameters/always_kmsg_dump - - - - Y
+#w /sys/module/kernel/parameters/crash_kexec_post_notifiers - - - - Y
```

Comment:
add the cleanup rules for the pstore storage directory somewhere here, too

Reply:
done